src/TodoList.js

```diff
@@ -1,18 +1,11 @@
 import React from "react"
 import TodoListItem from "./TodoListItem"
-import App from "./App"
 
-var todoList = [
-    <App props="TodoList" />
-]
-function TodoList() {
+function TodoList(props) {
+
   return (
     <ul>
-        {
-        todoList.map((todo) => (
-          <TodoListItem todo={todo} />
-        ))
-        }
-    </ul>
-  )
-}
\ No newline at end of file
+      {props.todoList.map((todo) => (
+        <TodoListItem todo={todo} />
```

Comment:
every time you map through the list and return an item you want this item to have a unique key, this way React knows and identify which items have changed